src/govee_local_api/light_capabilities.py

```diff
@@ -212,6 +212,8 @@ def create_with_capabilities(
     "H7033": BASIC_CAPABILITIES,
     "H70C1": BASIC_CAPABILITIES,
     "H70C2": BASIC_CAPABILITIES,
+    "H70C4": create_with_capabilities(True, True, True, 10, True),
+    "H70C5": create_with_capabilities(True, True, True, 10, True),
```

Comment:
🤔 Verified with the app that my lights `H70C5` has ten segments.
Not sure for the `H70C4` since I don't have it. 
I don't know if we could taken the info from somewhere, on the website this info seems not mentioned.

Reply:
Hi,
 first of all, thanks for the PR
In this case I would leave the H70C4 with the segments control disabled (segments count to 0).

Reply:
Hi,
I have H70C4 and I can control 10 segments through an app
![IMG_1579](https://github.com/user-attachments/assets/18a1c981-6d76-444e-98f2-ce188c255b36)

Reply:
Nice @matiqos!
@Galorhallen so previous commit dropped 😅